core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -966,8 +970,8 @@ public MatchImplementor get(final SqlMatchFunction function) {
     }
   }
 
-  public TableValuedFunctionCallImplementor get(final SqlWindowTableFunction operator) {
-    final Supplier<? extends TableValuedFunctionCallImplementor> supplier =
+  public TableFunctionCallImplementor get(final SqlWindowTableFunction operator) {
+    final Supplier<? extends TableFunctionCallImplementor> supplier =
```

Comment:
Also remove the values keyword from the commit message.

Reply:
Done.